library/src/scala/tasty/util/ShowExtractors.scala

```diff
@@ -2,22 +2,19 @@ package scala.tasty.util
 
 import scala.tasty.Tasty
 
-class TastyPrinter[T <: Tasty with Singleton](val tasty: T) {
+class ShowExtractors[T <: Tasty with Singleton](tasty0: T) extends Show[T](tasty0) {
```

Comment:
I hope the name `ShowExtractors` can be improved later if possible.

Reply:
Yes